lib/Sema/ConstraintGraph.cpp

```diff
@@ -27,7 +27,9 @@
 using namespace swift;
 using namespace constraints;
 
+#if !defined(_MSC_VER)
 #pragma mark Graph construction/destruction
+#endif
```

Comment:
@hughbe Can you just globally disable the "unknown pragma" warning in the build using "-wd#"?